src/app/services/ccd-config/launch-darkly-defaults.constants.ts

```diff
@@ -1,145 +1,154 @@
 import { DeploymentEnvironmentEnum } from '../../enums/deployment-environment-enum';
+import { WAFeatureConfig } from '../../../work-allocation/models/common/service-config.model';
 
 // Temporary solution to the problem that LaunchDarkly initialisation doesn't complete before we
 // need to initialise ccd-case-ui-toolkit.
 // The full fix is to resolve the race condition so that toolkit initialisation
 // doesn't start until LD is fully initialised
 export class LaunchDarklyDefaultsConstants {
```

Comment:
I know this is a temp solution but if someone changes the LD config we should make them aware that they need to change this ideally so we do not get any other problems like this

Reply:
That was already in place, and this PR updates the temporary config to match the latest